lib/Runtime/PlatformAgnostic/Platform/Linux/NumbersUtility.cpp

```diff
@@ -0,0 +1,267 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Common.h"
+#include "ChakraPlatform.h"
+#include <locale.h>
+
+namespace PlatformAgnostic
```

Comment:
Yes and No. At the moment, we have some things from ICU, rest not. Although header file is common (hence the reason for ENABLE_GLOBALIZATION ifdef), I don't see a reason to compile non-ICU source file on ICU build.